consensus/polybft/state_store_governance.go

```diff
@@ -0,0 +1,183 @@
+package polybft
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/contractsapi"
+	"github.com/0xPolygon/polygon-edge/helper/common"
+	bolt "go.etcd.io/bbolt"
+)
+
+var (
+	governanceEventsBucket              = []byte("governanceEvents")
+	clientConfigBucket                  = []byte("clientConfig")
+	clientConfigKey                     = []byte("clientConfigKey")
+	lastProcessedGovernanceEventsBucket = []byte("lastProcessedGovernanceEvents")
+	lastProcessedGovernanceBlockKey     = []byte("lastProcessedGovernanceBlockKey")
```

Comment:
I'd suggest to consolidate naming, because `lastProcessedGovernanceBlockKey` identifies `lastProcessedGovernanceEvents`. So I'd suggest to rename it `lastProcessedGovernanceBlock` and `lastProcessedGovernanceBlockKey` (or however you want, just for the clarity sake, pls add suffix `Key` to the bucket name).

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1749/commits/6bcffbbd344e5db00b863d22c1872ace2a23a9f6